vpd-tool/include/tool_utils.hpp

```diff
@@ -125,5 +125,63 @@ inline std::string getPrintableValue(const types::BinaryVector& i_keywordValue)
 
     return l_oss.str();
 }
+
+/**
+ * @brief API to read keyword's value from hardware.
+ *
+ * This API reads keyword's value by requesting DBus service(vpd-manager) who
+ * hosts the 'ReadKeyword' interface to read keyword's value.
```

Comment:
`ReadKeyword` is a method and not an interface? `ReadKeyword` method is on the `com.ibm.VPD.Manager` interface.

Reply:
updated